### ISSUE TITLE
Rethinking how non-jetpack banners display inside the jetpack dashboard

### ISSUE THREAD

**Issue by jeffgolenski** [Automattic/jetpack#10902]
Just taking a look at ManageWP and noticed that a lot of non-jetpack banners have a way of showing up within our dashboard page and possibly confusing customers.

If I didn't know better, I'd thinking this banner was a prompt from Jetpack:

<img width="1672" alt="screen shot 2018-12-09 at 4 17 21 pm" src="https://user-images.githubusercontent.com/214813/49703087-9cf5f680-fbce-11e8-8a13-9bb0fd3ed708.png">

I'd like to entertain thinking of ways to minimize the possible confusion here. Perhaps creating a designated "non jetpack" display area for all wp-admin banners that aren't ours. 

cc @joanrho @keoshi

**Comment by rickybanister** [comment]
Since we have created a 'single-page-app' within Jetpack's dashboard, it feels appropriate to localize all banners and things to our app and to hide all others. I don't want to break any .org rules, but that feels right to me.

@jessefriedman @beaulebens do you concur?

**Comment by jessefriedman** [comment]
@rickybanister I concur completely.  I don't think there are any rules that say we can't suppress other plugins' banners on our plugin pages. However, if we feel that is something we need to check-in on I can ping Mika.  

cc/  @beaulebens 

@jeffgolenski  good catch

**Comment by beaulebens** [comment]
I agree; I think it makes sense to suppress any "unknown" banners on this view since it's almost certain that they're not well-targeted (unless they're actively malicious!) and are just "overflowing" from being set to display everywhere in the admin.
